bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -318,6 +328,42 @@ impl Connection {
         Ok(self)
     }
 
+    /// Set a custom callback function which is run during client certificate validation during
+    /// a mutual TLS handshake.
```

Comment:
This isn't limited to mutual TLS, as this alludes to. It's just called any time a certificate needs to be validated. This _does_ include client cert verification on the server.